packages/editor/src/hooks/pattern-overrides.js

```diff
@@ -45,7 +44,6 @@ const withPatternOverrideControls = createHigherOrderComponent(
 				{ props.isSelected && isSupportedBlock && (
 					<ControlsWithStoreSubscription { ...props } />
 				) }
-				{ isSupportedBlock && <PatternOverridesBlockControls /> }
```

Comment:
The crux: removes the need for this component hooking into all editable blocks (performance issue) by moving it into the Block Toolbar